subprojects/distributions/src/integTest/groovy/org/gradle/DistributionIntegrationSpec.groovy

```diff
@@ -34,6 +34,7 @@ abstract class DistributionIntegrationSpec extends AbstractIntegrationSpec {
     @Rule public final PreconditionVerifier preconditionVerifier = new PreconditionVerifier()
 
     @Shared String version = GradleVersion.current().version
+    @Shared String baseVersion = GradleVersion.current().baseVersion.version
```

Comment:
"baseVersion" makes sense to me (since 'base' is a more specific version).  That's also what we call it in GradleVersion